packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -481,144 +451,154 @@ export class DataMenu extends React.PureComponent {
+                ))
+              : null}
+          </MenuGroup>
+        </MenuItem>
       </RestrictedMenuItem>
     );
   };
 
+  renderSupportLink = () => {
+    const itemIndex = ['fixed', 'support'];
+    const isAlternativeTheme = true;
+    const isActive = this.state.activeItemIndex === itemIndex.join('-');
+    return (
+      <MenuItem
+        hasSubmenu={false}
+        isActive={false}
+        isMenuOpen={this.state.isMenuOpen}
+        onClick={() => {
+          this.handleToggleItem(...itemIndex);
+        }}
+        onMouseEnter={
+          this.state.isMenuOpen
+            ? null
+            : () => this.handleToggleItem(...itemIndex)
+        }
+        onMouseLeave={this.state.isMenuOpen ? null : this.shouldCloseMenuFly}
+      >
+        <a href={MCSupportFormURL} rel="noopener noreferrer" target="_blank">
+          <div className={styles['item-icon-text']}>
+            <div className={styles.icon}>
+              <SupportIcon
+                size="scale"
+                theme={getIconTheme(isActive, isAlternativeTheme)}
+              />
+            </div>
+            <div className={styles.title}>
+              <FormattedMessage {...messages['NavBar.MCSupport.title']} />
+            </div>
+          </div>
+        </a>
+      </MenuItem>
+    );
+  };
+
   render() {
     return (
       <MenuGroup level={1}>
-        <ScrollableMenu>
-          {this.props.data
-            .filter(menu => this.bottomMenuItems.indexOf(menu.key))
-            .map((menu, index) => this.renderMenu(menu, 'scrollable', index))}
-        </ScrollableMenu>
-        <FixedMenu>
-          {this.props.data
-            .filter(menu => !this.bottomMenuItems.indexOf(menu.key))
-            .map((menu, index) => this.renderMenu(menu, 'fixed', index))}
+        <div className={styles['scrollable-menu']}>
+          {this.props.data.map((menu, index) => (
+            <React.Fragment key={index}>
+              {menu.shouldRenderDivider && <MenuItemDivider />}
+              {this.renderMenu(menu, 'scrollable', index)}
+            </React.Fragment>
+          ))}
+        </div>
+        <div className={styles['fixed-menu']}>
+          {this.renderSupportLink()}
           <MenuExpander
             isVisible={this.state.isExpanderVisible}
             onClick={this.handleToggleMenu}
           />
-        </FixedMenu>
+        </div>
       </MenuGroup>
     );
   }
 }
 
-export const ScrollableMenu = props => (
-  <div className={styles['scrollable-menu']}>{props.children}</div>
-);
-ScrollableMenu.displayName = 'ScrollableMenu';
-ScrollableMenu.propTypes = {
-  children: PropTypes.node,
-};
-
-export const FixedMenu = props => (
-  <div className={styles['fixed-menu']}>{props.children}</div>
-);
-FixedMenu.displayName = 'FixedMenu';
-FixedMenu.propTypes = {
-  children: PropTypes.node,
-};
-
-export const NavBarLayout = props => (
+export const NavBarLayout = React.forwardRef((props, ref) => (
```

Comment:
NOTE that we use `React.forwardRef`, as documented here: https://reactjs.org/docs/forwarding-refs.html